lisa/tests/test_env_requirement.py

```diff
@@ -17,7 +17,12 @@
     generate_min_capability,
 )
 from lisa.tests.test_search_space import SearchSpaceTestCase
-from lisa.testsuite import DEFAULT_REQUIREMENT, TestCaseRequirement, simple_requirement
+from lisa.testsuite import (
```

Comment:
The commit description "Update UT" is not clear. The "Update UT" explains what does, but it's obvious. It should explain the reason to change, or something is not obvious. For example, "Fix UT which is failed by simple_requirement refactoring". But we should try to make every commit works, including UT. So the UT fix should be in the same commit of the requirement change.